t/005-exit.t

```diff
@@ -491,7 +491,7 @@ GET /lua
 --- config
     location /lua {
         content_by_lua '
-            ngx.exit(ngx.HTTP_METHOD_NOT_IMPLEMENTED)
+            ngx.exit(ngx.HTTP_NOT_IMPLEMENTED)
```

Comment:
Should keep the old test to make sure backward compatibility. Better add a new one.